CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributing guidelines
+
+Thank you for your interest in the project. We look forward to your
```

Comment:
I actually decided against it when writing this document because I feel like the major part is irrelevant to what we're doing here. I'd rather refer to specific subsections like we do in the "sign your commits" part. Do you feel like there's one part of the docker/docker document that should be explicitly mentioned but isn't?